Front/js/APIpaths.js

```diff
@@ -1,19 +1,14 @@
-require('dotenv').config();
-const domain = `http://localhost:${process.env.PORT}`
+const domain = `http://localhost:3000`;
```

Comment:
need to check how it can work with .env
old version was:
require('dotenv').config();
const domain = `http://localhost:${process.env.PORT}`